### DISCUSSION TITLE
Mac error - Topotoolbox

### DISCUSSION THREAD

**Discussion by amariph** [wschwanghart/topotoolbox#31] [Q&A]
Hi! 

I want to ask if there is a Mac user that has the same error that I have when I try to run some functions. I thought it was only me, but it seems it is something that happens to another colleague with a Mac Computer.  We don’t know there is something with the compiler. 

<img width="255" alt="Screen Shot 2023-02-12 at 7 31 37 PM" src="https://user-images.githubusercontent.com/47638951/223856311-1093cdc2-6462-4526-b021-5d59469a8401.png">

**Comment by wschwanghart** [comment]
Hi, I am not a Mac user, but I had a similar issue in a course recently, and I was unable to resolve it. I actually deleted all compiled mex-files, but even then it wouldn't work. I hope others might be able to help out here.

**Reply by wschwanghart** [reply]
Martin Trauth pointed me to a question on MATLAB answers. Does that help: https://de.mathworks.com/matlabcentral/answers/516152-mex-file-cannot-be-opened-because-the-developer-cannot-be-verified-in-os-x-10-15-2-and-cannot-be-whi

**Comment by wschwanghart** [comment]
If you want a quick work-around until we have fixed this, you'll need to make sure that no mex-files are called. For example, in the function [flowacc.m](https://github.com/wschwanghart/topotoolbox/blob/master/%40FLOWobj/flowacc.m), change line 68
`if ~(exist(['flowacc_mex.' mexext],'file') == 3 && nargin<3 && strcmp(FD.type,'single'))`
to
`if true %~(exist(['flowacc_mex.' mexext],'file') == 3 && nargin<3 && strcmp(FD.type,'single'))`

**Reply by amariph** [reply]
Thank you so much professor Wolfgang!!! It seems that is working!!! 

Thanks again!!
